hw4Task3.js

```diff
@@ -0,0 +1,17 @@
+let minAge = 18
+let maxAge = 60
+let age = Number('60')
+
+if (isNaN(age) && age !== '') {
+    console.log('Incorrect data type')
+} 
```

Comment:
В данном задании не приводим к числу до проверки `isNaN`, она для єтого и нужна
```suggestion
let age = '60';

if (isNaN(age)) {
    console.log('Incorrect data type')
} 
```